charts/sonarqube/Chart.yaml

```diff
@@ -21,10 +21,8 @@ annotations:
     - name: support
       url: https://community.sonarsource.com/
   artifacthub.io/changes: |
-    - kind: added
-      description: "added link to community support forum"
-    - kind: changed
-      description: "Use liveness endpoint instead of helth endpoint for liveness probe"
+    - kind: fixed
```

Comment:
The previous changes have not been released yet, so please keep this list additive

Reply:
Ahh, ok, shall I then also keep chart version as 1.1.10? I created a 1.1.11 version for my change. Should I also change that back to version 1.1.10?

Reply:
nah the version increment to 1.1.11 (nice version btw :D) is correct

Reply:
ok, I have added back the changes to the Chart.yaml file